backend/app/api/endpoints/adapter/tasks.py

```
@@ -684,3 +684,213 @@ def delete_task_services(
+    try:
+        async with httpx.AsyncClient(timeout=120.0) as client:
+            response = await client.get(f"{base_url}/files/download-zip")
+
+            if response.status_code != 200:
+                logger.error(
+                    f"Failed to download workspace files: HTTP {response.status_code}"
+                )
+                raise HTTPException(
+                    status_code=response.status_code,
+                    detail=f"Failed to download workspace files: {response.text[:200]}",
+                )
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)